scripts/es.js

```diff
@@ -17,12 +17,12 @@
  * under the License.
  */
 
+require('../src/setup_node_env');
```

Comment:
__note:__ not sure if it's the correct solution - but I wanted to TypeScript in `kbn/es`. Let me know if there is a better solution or if you prefer me to switch to JavaScript instead.

Reply:
That's fine, as long as it doesn't have to be transpiled.